resources/ts/Components/FormDropdown.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from "react";
+import { Heading } from "./Heading";
+import { ButtonIcon } from "./ButtonIcon";
+import {
+  faPencil,
+  faChevronRight,
+  faChevronDown,
+} from "@fortawesome/free-solid-svg-icons";
+type Props = {
+  children: React.ReactNode;
+  title: "string";
+  setIsDisabled: React.Dispatch<React.SetStateAction<boolean>>;
+};
+
+export const FormDropdown = ({ title, children, setIsDisabled }: Props) => {
```

Comment:
Passing setters like here `setIsDisabled` is a bas practice. It is better to pass 2 props:
- disabled
- onChangeDisabled

Reply:
I'm also wondering.

This disabled state is to collapse and uncollapse right?

Then naming it `isCollapsed` and `setIsCollapsed` seems more correct 🙂

Reply:
But I also see
`const [isVisible, setIsVisible] = useState(false);`

So that could also be your collapsed, which is perfectly fine 👍🏻